source/dem/dem.cc

```diff
@@ -191,6 +202,8 @@ DEMSolver<dim, PropertiesIndex>::setup_distribution_type()
                 lpp.seed_for_distributions[particle_type] + this_mpi_process);
             break;
         }
+      // size_distribution_object_container[particle_type]
+      //   ->print_psd_declaration_string(particle_type, pcout);
```

Comment:
```suggestion
// To be uncommented in a follow-up PR since this will change all test outputs.
      // size_distribution_object_container[particle_type]
      //   ->print_psd_declaration_string(particle_type, pcout);
```